rmgpy/molecule/draw.py

```diff
@@ -864,8 +864,11 @@ def __generateAtomLabels(self):
         self.symbols = symbols = [atom.symbol for atom in atoms]
         for i in range(len(symbols)):
             # Don't label carbon atoms, unless there are only one or two heavy atoms
+            # or they are isotopically labeled
             if symbols[i] == 'C' and len(symbols) > 2:
-                if len(atoms[i].bonds) > 1 or (atoms[i].radicalElectrons == 0 and atoms[i].charge == 0):
+                if (len(atoms[i].bonds) > 1 or \
+                        (atoms[i].radicalElectrons == 0 and atoms[i].charge == 0))\
+                        and atoms[i].element.isotope== -1:
```

Comment:
So this implies that it will draw the letter of any isotopic element along with coloring it green?

Reply:
Exactly! So for a long linear hydrocarbon, where typically no letters are created, if a carbon is enriched, the algorithm will print the 'CH2' and color the 'CH2' green, so the user can notice that the enrichment is in that part of the molecule. If the letter is typically drawn anyways, then the only change is coloring it green.

Reply:
Ok